source/manage-apps/configure/config.txt

```diff
@@ -38,6 +38,16 @@ using structured JSON configuration and JavaScript source code files.
 - ``.js`` files define serverless application logic used in functions, triggers,
   HTTPS endpoints, and custom resolvers.
 
+.. important:: Reserved File Names
+   
+   You can include arbitrarily named files in your realm app config, with a few exceptions.
+   Note the following names: ``stitch.json``, ``config.json``, 
+   ``realm_config.json``, and ``app_config.json``. These names are reserved specific 
+   files used to define and configure specific components in your app. As such, files 
+   of these names must conform to a specific schema for its component type. 
+   Using these file names when the file's content fails to conform to that schema
+   will throw an error when you import the app using the CLI.
+   
```

Comment:
how about another sentence here w a link to where the user can learn more abt configuring these files? i think this page would be a good place to link to - https://www.mongodb.com/docs/atlas/app-services/manage-apps/configure/config/

Reply:
i agree about having a reference, but isn't the link you sent just the same page we're editing now?